images/nginx/rootfs/build.sh

```diff
@@ -364,8 +368,9 @@ make install
 
 # build modsecurity library
 cd "$BUILD_PATH"
-git clone https://github.com/SpiderLabs/ModSecurity
+git clone --depth=100 https://github.com/SpiderLabs/ModSecurity
```

Comment:
why depth 100?

Reply:
Because we are not using a tag, the commit we have now is ~67 behind master and we cannot use the last commit because of a refactor the module is having.
The reason of the depth is to speed up the build for arm,arm64, and s390x.